### ISSUE TITLE
Update docker-compose.yml file version

### ISSUE THREAD

**Issue by riggraz** [astuto/astuto#367]
Running `docker compose up` returns the following warnings:

```
WARN[0000] /Users/riccardo/Documents/me/astuto/astuto/docker-compose.yml: `version` is obsolete 
WARN[0000] /Users/riccardo/Documents/me/astuto/astuto/docker-compose.override.yml: `version` is obsolete 
```

**Comment by rishabhj1717** [comment]
Hey @riggraz , I would like to contribute and take this issue up.

**Comment by riggraz** [comment]
Hi @rishabhj1717, that'd be great! Thanks for your help!

If you have time, you may also want to check out another issue related to docker-compose.yml file: #167 

Let me know if you have any questions :)

**Comment by rishabhj1717** [comment]
@riggraz Sorry for the delayed response, but yes, will check out the other issue too and pick it up once this issue is done 👍
